go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/compose-spec/compose-go
 go 1.16
 
 require (
-	github.com/compose-spec/godotenv v1.0.0
+	github.com/compose-spec/godotenv v0.0.0-20211026074040-865524795bdc
```

Comment:
There has just been a v1.1.0 of `compose-spec/godotenv` that’s been released. 😁

Reply:
Nice, thx, updated the version to v1.1.0